### ISSUE TITLE
Bluetooth issue

### ISSUE THREAD

**Issue by Rafiq226** [itsMeDavidV/Watchla-Support#151]
Can not connect to Bluetooth from the watch with the app.

**Comment by Rafiq226** [comment]
Can not connect Bluetooth from Watchla app.

**Comment by itsMeDavidV** [comment]
Hi @Rafiq226 can you please send your Watchla support code via https://watchla.app/contact so this can be investigated? 

Thanks,
-David

**Comment by Rafiq226** [comment]
How do you obtain support code?

**Comment by JalinEnterprisesLLC** [comment]
Same with me - No change with latest 3.4.6 update.
New support code:
WyTDSY3D4o

**Comment by arcateq** [comment]
Cannot connect to Bluetooth as well just hangs on initializing 

nC1s2tAJAP

Un able to connect Bluetooth
Hangs on initialing on the watch.

WATCH: Apple Watch Ultra 9.6
PHONE: iPhone 14 Pro Max 16.6
APP: 3.4.6 (294)
TESLA:MYLR2023 2023.26.7
nC1s2tAJAP

**Comment by JalinEnterprisesLLC** [comment]
Please add me to the request for Beta - Thanks

**Comment by JalinEnterprisesLLC** [comment]
After loading Beta 3.6.7 still get either scanning or initializing when trying to add Bluetooth digital key:
Support code: HCGLRtgfMZ
Apple Watch series 6 - latest os
Tesla Model (2018) S/W 2023.26.7
Fyi - I then reloaded my no longer used Watch for Tesla App and it initially had the same issue, but when i upgraded that to their latest beta version 1.3.10 it was able to add the key

**Comment by itsMeDavidV** [comment]
Hi @JalinEnterprisesLLC can you try again after deleting and reinstalling the Watchla watch app?

**Comment by JalinEnterprisesLLC** [comment]
I did that once, but will try that again and report back

Thanks,
Chris

**Comment by itsMeDavidV** [comment]
Thanks! @JalinEnterprisesLLC

**Comment by Rafiq226** [comment]
Hi David,
I have sent the Support code. Please confirm and update. I have reinstalled the app but still the same issue.
Thanks,
Rafiq

**Comment by Rafiq226** [comment]
Hi David,
I have sent the Support code. Please confirm and update. I have reinstalled
the app but still the same issue.
Thanks,
Rafiq

**Comment by JalinEnterprisesLLC** [comment]
I deleted the app, reboot of iPhone 11 and then reinstalled Watchla again.  Still no go, but I noticed that even after deleting app it still didn’t make me re login to Tesla when I reinstalled the app, so I logged out, deleted app and rebooted iPhone again.  The Watchla app did have me log into Tesla this time, but no go / still “scanning” and no adding the Digital key.  Any other suggestions?Really don’t want to reset iPhone On Aug 12, 2023, at 6:46 PM, David Villegas ***@***.***> wrote:﻿
Thanks! @JalinEnterprisesLLC

**Comment by itsMeDavidV** [comment]
@JalinEnterprisesLLC no need to reset. We're waiting on Apple to approve another update, will update then.

**Comment by itsMeDavidV** [comment]
@Rafiq226 got your email! Have you updated to v3.4.7?

**Comment by itsMeDavidV** [comment]
@arcateq Do you have a different support code on your watch? If not can you go ahead and restart the watch app and try setting up Bluetooth again? Also can you confirm that your watch app is on v3.4.7? This can be checked at the bottom of preferences on your watch app. Thanks!

**Comment by JalinEnterprisesLLC** [comment]
psDWm5JOzQOn Aug 12, 2023, at 8:46 PM, David Villegas ***@***.***> wrote:﻿
@arcateq Do you have a different support code on your watch? If not can you go ahead and restart the watch app and try setting up Bluetooth again? Thanks!

**Comment by arcateq** [comment]
Yes, still does not workSent from my iPhoneOn Aug 12, 2023, at 5:43 PM, David Villegas ***@***.***> wrote:﻿
@Rafiq226 got your email! Have you updated to v3.4.7?

**Comment by arcateq** [comment]
DpSsu47op2Sent from my iPhoneOn Aug 12, 2023, at 5:55 PM, JalinEnterprisesLLC ***@***.***> wrote:﻿
psDWm5JOzQOn Aug 12, 2023, at 8:46 PM, David Villegas ***@***.***> wrote:﻿

@arcateq Do you have a different support code on your watch? If not can you go ahead and restart the watch app and try setting up Bluetooth again? Thanks!

**Comment by arcateq** [comment]
DpSsu47op2Sent from my iPhoneOn Aug 12, 2023, at 5:46 PM, David Villegas ***@***.***> wrote:﻿
@arcateq Do you have a different support code on your watch? If not can you go ahead and restart the watch app and try setting up Bluetooth again? Thanks!

**Comment by itsMeDavidV** [comment]
Unfortunately there still seems to be a small amount of people who still face this issue. Watchla will be releasing v3.4.8 which has prioritized a different way of identifying a Tesla via Bluetooth. This is a very new way for the 3rd party community (as in this was discovered literally today and has not been documented yet) so there will be some validation needed. 

Apple is currently reviewing v3.4.8 beta, however since it is Saturday night in California right now, this process might be delayed.

If you would like to help validate v3.4.8 beta and have not requested beta, please do so.

**Comment by itsMeDavidV** [comment]
Hi everyone, the v3.4.8 update is now available on the [App Store](https://apps.apple.com/us/app/watchla-for-tesla/id1546599747) and it should fix this issue for you.

Before testing please ensure that your *watch* app is up to date. You can check which version the watch app is running by going to Preferences in the Watch app and scrolling all the way down.

Thank you again for your patience.

-David

**Comment by Rafiq226** [comment]
Hi David,
It's working now. Bluetooth is connecting as expected. Thanks a lot for
your help.

Rafiq

**Comment by itsMeDavidV** [comment]
@Rafiq226 Sweet! That's really good to hear. I'm going to wait to close this issue until tomorrow incase others who are still on older version run into the same issue.

**Comment by JalinEnterprisesLLC** [comment]
The only negative thing I have noticed (will see if it continues or is just a coincidence) is the walk away lock wasn’t working reliably after the update, and I never had issues with your app before with this. If it continues I will do a couple things like rebooting and/or uninstall/reinstall etc..Thanks again for the fix and hopefully this will not be a recurring issue with future Tesla updates.On Aug 14, 2023, at 1:28 PM, David Villegas ***@***.***> wrote:﻿
@Rafiq226 Sweet! That's really good to hear. I'm going to wait to close this issue until tomorrow incase others who are still on older version run into the same issue.
